src/test/components/DepositWidget.test.ts

```diff
@@ -0,0 +1,11 @@
+import { getButtonText, ButtonTypes } from '../../components/DepositWidget/Row'
+
+test('Deposit button text', () => {
+  expect(getButtonText(ButtonTypes.DEPOSIT, '')).toMatch(/Deposit/)
```

Comment:
Why do you use regex?
it's not enough to use a string `toBe('Deposit')`

Reply:
Mainly because right now there are `-` and `+` signs that, from the mocks, might be an img or another element for styling later on. Otherwise I can use `toBe`

Reply:
Okay :)